templates/style/_navbar.scss

```diff
@@ -28,6 +28,12 @@ div.nav-container {
     color: var(--color-navbar-standard);
     /* The font size must be specified in pixels because the height is specified in pixels. */
     font: 16px $font-family-sans;
+    position: relative;
+
+    .container, .pure-menu-horizontal {
+        position: relative;
+        height: 100%;
```

Comment:
What are the `position: relative` and `height: 100%` needed for here? I think they are unnecessary.

Reply:
It's because with this, we can set the element of the navbar to `height: 100%`. It prevents the bottom margin to disappear in case the font size is too big.